### DISCUSSION TITLE
Lesson 13: onClick activateBrowserWallet not working

### DISCUSSION THREAD

**Discussion by toddabraham** [smartcontractkit/full-blockchain-solidity-course-py#252] [Q&A] [answered]
Hi there,

![image](https://user-images.githubusercontent.com/26919212/138579472-914ce031-5289-44e3-bcbb-4c861bdf5b6e.png)

I have the button showing up as per the screen shot above, but my MetaMask Wallet is not opening on click.

Here's my Header.tsx code....

```js
import { useEthers } from "@usedapp/core"

export const Header = () => {
    const { account, activateBrowserWallet, deactivate } = useEthers()

    const isConnected = account !== undefined

    return (
        <div>
            {isConnected ? (
                <button color="primary">
                    onClick={deactivate}
                    Disconnect
                </button>
            ) : (
                <button color="primary"
                    onClick={() => activateBrowserWallet
                    }>
                    Connect
                </button>
            )
            }
        </div>
    )
}
```

**Comment by toddabraham** [comment, marked as answer]
Found out I had to manually connect `http://localhost:3000/` to my browser MetaMask Wallet.

![image](https://user-images.githubusercontent.com/26919212/138620005-016b2374-9886-451a-96c2-f9c6602bddac.png)

![image](https://user-images.githubusercontent.com/26919212/138620069-b1fa7978-624b-47ae-8742-e716ab8369e9.png)

Hope this is helpful for someone.

**Reply by shonendumm** [reply]
This is helpful for me, thank you!
